src/backend/marsha/bbb/tests/test_models.py

```diff
@@ -0,0 +1,391 @@
+    def test_start_new_meeting(self, mock_create_request):
+        """Create a meeting in current meeting related server."""
+        meeting = MeetingFactory(
+            attendee_password="9#R1kuUl3R",
+            moderator_password="0$C7Aaz0o",
+        )
+        mock_create_request.return_value.status_code = 200
+        mock_create_request.return_value.content = f"""
+        <response>
+            <returncode>SUCCESS</returncode>
+            <meetingID>{meeting.id}</meetingID>
+            <internalMeetingID>232a8ab5dbfde4d33a2bd9d5bbc08bd74d04e163-1628693645640</internalMeetingID>
+            <parentMeetingID>bbb-none</parentMeetingID>
+            <attendeePW>{meeting.attendee_password}</attendeePW>
+            <moderatorPW>{meeting.moderator_password}</moderatorPW>
+            <createTime>1628693645640</createTime>
+            <voiceBridge>83267</voiceBridge>
+            <dialNumber>613-555-1234</dialNumber>
+            <createDate>Wed Aug 11 14:54:05 UTC 2021</createDate>
+            <hasUserJoined>false</hasUserJoined>
+            <duration>0</duration>
+            <hasBeenForciblyEnded>false</hasBeenForciblyEnded>
+            <messageKey></messageKey>
+            <message></message>
+        </response>
+        """
+
+        api_response = meeting.start()
+        self.assertDictEqual(
+            {
+                "attendeePW": meeting.attendee_password,
+                "createDate": "Wed Aug 11 14:54:05 UTC 2021",
+                "createTime": "1628693645640",
+                "dialNumber": "613-555-1234",
+                "duration": "0",
+                "hasBeenForciblyEnded": "false",
+                "hasUserJoined": "false",
+                "internalMeetingID": "232a8ab5dbfde4d33a2bd9d5bbc08bd74d04e163-1628693645640",
+                "meetingID": str(meeting.id),
+                "message": "Meeting created.",
+                "messageKey": None,
+                "moderatorPW": meeting.moderator_password,
+                "parentMeetingID": "bbb-none",
+                "returncode": "SUCCESS",
+                "voiceBridge": "83267",
+            },
+            api_response,
+        )
+        # self.assertEqual(meeting.started, True)
+
+    @mock.patch.object(requests, "get")
+    def test_start_existing_meeting(self, mock_create_request):
+        """Create a meeting in current meeting related server."""
+        meeting = MeetingFactory()
+        mock_create_request.return_value.status_code = 200
+        mock_create_request.return_value.content = """
+        <response>
+            <returncode>FAILED</returncode>
+            <messageKey>idNotUnique</messageKey>
+            <message>
+                A meeting already exists with that meeting ID.
+                Please use a different meeting ID.
+            </message>
+        </response>
+        """
+
+        with self.assertRaises(ApiMeetingException):
+            meeting.start()
+            self.assertEqual(meeting.started, True)
```

Comment:
If the expection is raised, I don't understand how `meeting.started` can be true. IMO this line should not be in the `with` block. WDYT ?